### PR TITLE
nodemon Setup

### PR DESCRIPTION
**Pull request by Bhavesh27**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,11 +11,12 @@
   },
   "devDependencies": {
     "env-cmd": "^10.1.0",
-    "ngrok": "^3.3.0"
+    "ngrok": "^3.3.0",
+    "nodemon": "^2.0.4"
```
Comment (mukarramali) [inline package.json:15]: `yarn.lock` isn't updated. Please add that too. Everything else looks good

**Review by mukarramali — CHANGES_REQUESTED** [review]

**Comment by mukarramali** [comment]
Please link issues you are working on

**Comment by mukarramali** [comment]
@Bhavesh27 it has merge conflicts.

File: package.json (hunk 1 above)
Comment (Bhavesh27) [reply package.json:18]: Resolved

**Review by mukarramali — APPROVED** [review]

**Pull request merged**